02/app.js

```diff
@@ -1,7 +1,23 @@
 
-/* rozwiązanie z pętlą for */
-const x = 4;
+/* rozwiązanie z pętlą for 
+const x = prompt('Podaj liczbę!')
+if (x >= 1 && x <= 9) {
+for (i = 0; i <= 9; i++) {
+    const multiplication = x * i;
+    console.log(multiplication);
+}
+}
+else {
+    console.log("Źle! Podaj liczbe od 1 do 9")
+}
+*/
 
+/* rozwiązanie z pętlą while  */1
+const a = prompt('Podaj liczbę');
+const n = prompt('Podaj liczbę');
+const result = a ** n;
+while (a > 0 && n > 0) {
 
+console.log(result);
+}
```

Comment:
Chyba czegoś tu brakuje ;)

Reply:
Ahh, zle zrozumialem polecenie.

Reply:
Mam wrazenie, ze tu calkiem inaczej to rozwiazalem i albo nie zapisalem, albo nie wiem o co chodzi

Reply:
Ok mam z tym troche problem, nie wiem jak to zrobic, aby wynik wyswietlal sie w formie calego dzialania, tak jak w poleceniu.